packages/grid/src/GridUtils.test.ts

```diff
@@ -190,15 +188,38 @@ describe('start/end range adjustment in one dimension', () => {
     testRange(10, 20, movedItems);
   });
   it('handles items moved into the range', () => {
-    const movedItems = GridUtils.moveItem(100, 7);
-    testRange(5, 10, movedItems, [
+    testRange(5, 10, GridUtils.moveItem(1, 7), [
+      [6, 10],
+      [1, 1],
```

Comment:
I would've thought this would be `[6,6], [1,1], [7,10]` based on our discussion yesterday... If we're not guaranteeing the order (which we may not need to - if it's important then you can just getModelRangeIndexes with the specific indices that matter for you), then may as well sort the results so test cases are more consistent.